surveyor-core/src/Surveyor/Core/SymbolicExecution.hs

```diff
@@ -186,8 +188,9 @@ suspendedState :: forall sym arch s ext st fs m init reg p rtp f a
                -> SymbolicState arch s sym init reg
                -> CSET.SimState p sym ext rtp f a
                -> Maybe (SCB.Breakpoint sym)
+               -> Maybe (CT.ModelView)
```

Comment:
Would it make sense to have a type to combine the breakpoint and modelview?  Something like:
```
data SimulationData sym where
  Breakpoint :: SCB.Breakpoint sym -> SimulationData sym
  ModelView :: CT.ModelView -> SimulationData sym
```